lib/screens/domain/todos_entity.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:core';
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:equatable/equatable.dart';
+
+class TodosEntity extends Equatable {
+  final bool complete;
+  final String id;
+  final String note;
+  final String task;
+
+  const TodosEntity(this.task, this.id, this.note, this.complete);
+
+  Map<String, Object> toJson() {
+    return {
+      'complete': complete,
+      'task': task,
+      'note': note,
+      'id': id,
+    };
+  }
+
+  @override
+  String toString() {
+    return 'TodosEntity { complete: $complete, task: $task, note: $note, id: $id ,}';
+  }
+
+  static TodosEntity fromJson(Map<String, Object> json) {
```

Comment:
for parsing we using json_serializable package
remove this method and use package